### DISCUSSION TITLE
Untranslated texts/Make translations easier

### DISCUSSION THREAD

**Discussion by Atraphone** [zyrouge/symphony#156] [Ideas]
Hi, thanks for adding my translations and your app.

In new version (2023.04.80) have untranslated strings.

![Screenshot_20230402-215405.png](https://user-images.githubusercontent.com/106192957/229373327-dea9c9af-6226-4627-b6a5-428cc475213d.png)

![Screenshot_20230402-215347.png](https://user-images.githubusercontent.com/106192957/229373365-b78bcfb6-b819-4eae-99a3-101f7a81e96d.png)

![Screenshot_20230402-215415.png](https://user-images.githubusercontent.com/106192957/229373379-7a8ca232-0f3f-4831-861f-f464f98fe5aa.png)

![Screenshot_20230402-215456.png](https://user-images.githubusercontent.com/106192957/229373387-9d81bef1-2460-4489-a5c5-6a249e3cce27.png)

![Screenshot_20230402-215515.png](https://user-images.githubusercontent.com/106192957/229373400-df99fb67-471a-480d-a6b1-8e568dcf802e.png)

![Screenshot_20230402-215523.png](https://user-images.githubusercontent.com/106192957/229373413-aada63dc-3fd4-4023-aa6b-4f9d389b795b.png)

![Screenshot_20230402-215535.png](https://user-images.githubusercontent.com/106192957/229373417-f17769fe-e30d-4c96-9eb7-a40b60531e6f.png)

![Screenshot_20230402-224525.png](https://user-images.githubusercontent.com/106192957/229375526-4db8648e-d5bb-4e13-afef-785d38f3f2cb.png)

And one question: Is it possible to save localizations as XML files for further ease of translation of the application as in other projects?

**Comment by zyrouge** [comment]
> Is it possible to save localizations as XML files for further ease of translation of the application as in other projects?

In my opinion, YAMLs are easier than working with XMLs. Maybe it's time to switch to Weblate or Crowdin.

**Comment by phw** [comment]
@zyrouge if there is interest I can assist with setting up things with weblate. I'm also running my own weblate instance for some of my MusicBrainz related projects and would be offering Symphony to use this, too

**Comment by zyrouge** [comment]
Added a note for translation contributors on the [wiki](https://github.com/zyrouge/symphony/wiki/Translations-Guide#note-to-translation-contributors). The current mechanism avoids usage of third-party services at the cost of developer/user experience. But it might be a hassle to migrate away from it right now. Converting this to an discussion for further feedback.

**Comment by DomHeadroom** [comment]
Yes in my opinion it would be much easier for translator to use a weblate, it's free for open source projects too, and it does have handy stuff like automatic translator integration.

And it's much better to see which strings are missing from the translation by doing so.

And you let it modify the yaml file so you don't need to rely only on them to translate the app.

**Reply by zyrouge** [reply]
> Yes in my opinion it would be much easier for translator to use a weblate, it's free for open source projects too, and it does have handy stuff like automatic translator integration.
> 
> And it's much better to see which strings are missing from the translation by doing so.
> 
> And you let it modify the yaml file so you don't need to rely only on them to translate the app.

I am working on this, should be out in a couple of weeks.
